go/ct/st/code.go

```diff
@@ -142,3 +143,29 @@ func (c *Code) Copy() []byte {
 func (c *Code) String() string {
 	return fmt.Sprintf("%x", c.code)
 }
+
+// ToHumanReadableString returns a string with the length of the code and the
+// human readable form for the opcodes in range [start, start+length).
+// - If the slice to be printed overflows the existing code, the overlapping code is printed.
+// - If start exceeds the code length, the length of the code is printed.
+// - Data found in the code is printed as decimal numbers (to differentiate from unused opcodes).
+func (c *Code) ToHumanReadableString(start int, length int) string {
```

Comment:
nice addition !!! probably should have been a separate PR.

Reply:
yes.